lib/events/athena/querier.go

```diff
@@ -969,15 +965,13 @@ func (rb *responseBuilder) appendUntilSizeLimit(resultResp *athena.GetQueryResul
 				// page.
 				return true, nil
 			}
-			// A single event is larger than the max page size - the best we can
-			// do is try to trim it.
```

Comment:
I think this comment should stay here rather than moving to the trimToMaxSize impl. The purpose of this comment is to clarify that `len(eventData)` must be greater than `events.MaxEventBytesInResponse` because `len(eventData)+rb.totalSize > events.MaxEventBytesInResponse && len(rb.output) == 0`